### PR TITLE
Fix Github actions timeout errors

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Updated and new tests for non submisson Api endpoints

## Type of change

- New tests

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- Tested locally as well as on github actions

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: .github/workflows/cucumber-tests.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Cucumber Tests
 
 on:
   push:
-    branches: [master]
+    branches: [master, qa-updated-endpoints]
```
Comment (adodge-bcg) [inline .github/workflows/cucumber-tests.yml:5]: Does it still need qa-updated-endpoints?

**Review by adodge-bcg — APPROVED** [review]
Lg2m

File: .github/workflows/cucumber-tests.yml (hunk 1 above)
Comment (akroon3r) [reply .github/workflows/cucumber-tests.yml:5]: unless he's continually pushing changes here for future updates? I wouldn't be angry at this.

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/paymentpayload/PaymentPayload.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public static final String INTERNAL_CLIENT_NUMBER = "2234";
+    public static final String REDIRECT_URL = "http://redirect.com";
+    private GenerateCardUrlRequest generateCardUrlRequest;
+
+    public String validGenerateUrlPayload() throws IOException {
+        generateCardUrlRequest = new GenerateCardUrlRequest();
+        ObjectMapper objMap = new ObjectMapper();
+        return objMap.writeValueAsString(generateUpdateCard());
+    }
+    public GenerateCardUrlRequest generateUpdateCard(){
```
Comment (akroon3r) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/paymentpayload/PaymentPayload.java:19]:
```suggestion
    }
    
    public GenerateCardUrlRequest generateUpdateCard(){
```

File: .github/workflows/cucumber-tests.yml (hunk 1 above)
Comment (karthikmsqa) [reply .github/workflows/cucumber-tests.yml]: removed the branch from the flow.

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/AuthenticationPage.java [hunk 3]
```diff
@@ -36,17 +37,18 @@ public AuthenticationPage(WebDriver driver) {
...
 
     //Actions:
-    public void signInWithBceid(String userNm, String pwd) {
+    public void signInWithBceid(String userNm, String pwd) throws InterruptedException {
         wait = new WebDriverWait(driver, 90);
         wait.until(ExpectedConditions.titleIs("Government of British Columbia"));
         log.info("Waiting for the page to load...");
         userName.sendKeys(userNm);
         password.sendKeys(pwd);
+        Thread.sleep(2000L);
```
Comment (akroon3r) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/AuthenticationPage.java:46]: what is L?

File: tests/src/test/java/features/backendfeatures/1-generateUrlAndSubmissionForExistingGuid.feature [hunk 4]
```diff
@@ -46,16 +46,7 @@ Feature: When user uploads the documents, id is authenticated user details, navi
...
-    Given POST http request is made to "DOCUMENT_SUBMISSION" with valid existing CSO account guid and a single pdf file
-    When status code is 200 and content type is verified
-    Then verify submission id and document count is received
-    ## Call to generate url ##
-    Given POST http request is made to "GENERATE_URL_API" with client application, court details and navigation urls
-    When status code is 200 and content type is verified
-    Then verify expiry date and eFiling url are returned with the CSO account guid and submission id
-    ## call to set bambora ##
-
+      ## Call to submit document ##
```
Comment (akroon3r) [inline tests/src/test/java/features/backendfeatures/1-generateUrlAndSubmissionForExistingGuid.feature:49]: if this is commented and you don't need it, remove it please.

File: tests/src/test/java/features/backendfeatures/4-updateAndDeleteDocument.feature [hunk 5]
```diff
@@ -10,25 +10,25 @@ Feature: User can upload additional document and delete the documents
...
     Then verify submission id and document count is returned
     ## Call to get updated document wth filename ##
     Given "SUBMISSION" id with filename is submitted with GET http request
- #   Then validated status code is 200 and content type is not json
+    Then validated status code is 200 and content type is not json
     ## Call to update document properties ##
-    Given "SUBMISSION" id with payload is submitted to upload the document properties
-    When validated status code is 200 and content type
-   # Then verify document properties are updated
+  #  Given "SUBMISSION" id with payload is submitted to update the document properties
```
Comment (akroon3r) [inline tests/src/test/java/features/backendfeatures/4-updateAndDeleteDocument.feature:26]: same here

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/AuthenticationPage.java (hunk 3 above)
Comment (karthikmsqa) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/AuthenticationPage.java:46]: represents "Long" argument for milliseconds.

File: tests/src/test/java/features/backendfeatures/1-generateUrlAndSubmissionForExistingGuid.feature (hunk 4 above)
Comment (karthikmsqa) [reply tests/src/test/java/features/backendfeatures/1-generateUrlAndSubmissionForExistingGuid.feature:49]: This is not a commented out but an actual "comment" syntax in gherkin.

File: tests/src/test/java/features/backendfeatures/4-updateAndDeleteDocument.feature (hunk 5 above)
Comment (karthikmsqa) [reply tests/src/test/java/features/backendfeatures/4-updateAndDeleteDocument.feature:26]: Added explanation

**Review by akroon3r — APPROVED** [review]

**Pull request merged**